src/rules/color-named/README.md

```diff
@@ -105,3 +105,21 @@ a { color: $blue; }
 ```less
 a { color: @blue; }
 ```
+
+## Optional options
+
+### `ignoreProperties: ["/regex/", "string"]`
+
+Given:
```

Comment:
Can you please change this to:

----

For example, with `"never"`.

Given:

```js
["/^my-/", "composes"]
```

The following patterns are *not* considered warnings:

```css
a {
  my-property: red;
}
```

```css
a {
  my-other-property: red;
}
```

```css
a {
  composes: red from './index.css';
}
```

---